src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/component/CommunityAPIClient.kt

```diff
@@ -0,0 +1,26 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.component
+
+import mu.KLogging
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.stereotype.Component
+import org.springframework.web.reactive.function.client.WebClient
+import reactor.core.publisher.Mono
+
+@Component
+class CommunityAPIClient(
+  @Qualifier(value = "communityApiWebClient") private val communityAPIWebClient: WebClient,
+) {
+  companion object : KLogging()
+
+  fun makeRequest(uri: String, requestBody: Any) {
```

Comment:
maybe worth specifying this is a POST request.

Reply:
may also be worth noting that this is an async call. calls may expect this to raise an exception at the call site if it fails.

Reply:
Brill - all for readability